documentation/configuringDynaflowLauncher/DynaflowLaunchersInputFiles.tex

```diff
@@ -122,21 +122,27 @@ \subsection{Configuration file}
 \rowcolor{white}
 \multirow{-2}{*}{\small{OutputDir}}&  \multirow{-2}{*}{\small{string}} & \small{outputs will be written}  &  \multirow{-2}{*}{\small{current path}} \\
 \rowcolor{gray!10}
-\small{DsoVoltageLevel} & \small{real} & \small{Minimum voltage level of loads} & \small{45} \\
+\small{DsoVoltageLevel} & \small{real} & \small{Minimum voltage level of loads} & \small{45}
+\\\rowcolor{white}
+&  & \small{Maximum voltage for which} &  \\
```

Comment:
What do you think of "Maximum voltage level for which generator transformers are considered to be in the iidm file" ?